aos/closed_loop/cutOutDonutsWithWEP.ipynb

```diff
@@ -8,10 +8,10 @@
     "# CutOutDonuts with WEP\n",
```

Comment:
<p>Here the <code>initialCutoutPadding</code> is <code>5</code>.Yet below it says that it's <code>40</code> by default. So what is the default?     </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-ts/ts_analysis_notebooks/pull/39/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
<p>The text below needs to be updated.</p>